src/passes/OptimizeInstructions.cpp

```diff
@@ -459,6 +460,47 @@ struct OptimizeInstructions
         }
         // note that both left and right may be consts, but then we let
         // precompute compute the constant result
+      } else if (binary->op == LtSInt32) {
+        if (auto* c = binary->right->dynCast<Const>()) {
+          auto constValue = c->value.geti32();
+          if (constValue == 0) {
+            // this may increase size for compressed binaryes so it apply
+            // only for non-zero shrinkLevel levels
```

Comment:
```suggestion
            // this may increase size for compressed binaries
```

Please also say why it can help.

Reply:
Should I mention LLVM's lowered form and speed of runtimes in comments?